dapr/clients/grpc/client.py

```diff
@@ -31,6 +31,9 @@
     BulkStateItem,
 )
 
+from opencensus.ext.grpc import client_interceptor
```

Comment:
We can add `# type: ignore` to bypass type checking

for example: 
https://github.com/dapr/python-sdk/blob/69d237bb5b5cf94226649a2fd341446cffd33b59/examples/demo_actor/demo_actor/demo_actor_service.py#L5